### PR TITLE
issue(#1): Create buildpack for Aleo

### PR DESCRIPTION
**Pull request by lispking**
issue #1

### CODE REVIEW

File: aleo/detect.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+				Requires: []libcnb.BuildPlanRequire{
+					{Name: PlanEntryAleo},
+				},
+			},
+		},
+	}, nil
+}
+
+func (d Detect) aleoProject(appDir string) (bool, error) {
+	_, err := os.Stat(filepath.Join(appDir, "program.json"))
```
Comment (wangeguo) [inline aleo/detect.go:58]: Here you also have to check that the output of the previous compilation step is present

File: bin/main
Comment (wangeguo) [inline bin/main:1]: This file should not be submitted, ignored

File: bin/build
Comment (wangeguo) [inline bin/build:1]: This file should not be submitted, ignored

File: bin/detect
Comment (wangeguo) [inline bin/detect:1]: This file should not be submitted, ignored

File: buildpack.toml
Comment (wangeguo) [inline buildpack.toml:1]: Do  it need require `leo-dist`?

**Review by wangeguo — CHANGES_REQUESTED** [review]

File: buildpack.toml
Comment (lispking) [reply buildpack.toml:1]: not need

File: bin/detect
Comment (lispking) [reply bin/detect:1]: aha, miss .gitignore file

**Pull request merged**